src/uwtools/tests/drivers/test_mpas.py

```diff
@@ -0,0 +1,209 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+MPAS driver tests.
+"""
+import datetime as dt
+from pathlib import Path
+from unittest.mock import DEFAULT as D
+from unittest.mock import patch
+
+import f90nml  # type: ignore
+import pytest
+import yaml
+from pytest import fixture, raises
+
+from uwtools.drivers import mpas
+from uwtools.exceptions import UWConfigError
+from uwtools.scheduler import Slurm
+from uwtools.tests.support import fixture_path
+
+# Fixtures
+
+
+@fixture
+def cycle():
+    return dt.datetime(2024, 3, 22, 6)
+
+
+# Driver fixtures
+
+
+@fixture
+def config(tmp_path):
+    return {
+        "mpas": {
+            "execution": {
+                "executable": "atmosphere_model",
+                "batchargs": {
+                    "walltime": "01:30:00",
+                },
```

Comment:
Added and `walltime` and `platform` blocks to `MPAS` and `MPASInit` configs in order to simplify their `runscript()` tests.